integration/spark/src/main/scala/org/apache/spark/sql/secondaryindex/hive/CarbonInternalMetastore.scala

```diff
@@ -136,14 +137,16 @@ object CarbonInternalMetastore {
 
   def refreshIndexInfo(dbName: String, tableName: String,
       carbonTable: CarbonTable, needLock: Boolean = true)(sparkSession: SparkSession): Unit = {
-    val indexTableExists = CarbonInternalScalaUtil.isIndexTableExists(carbonTable)
+    val indexTableExists = CarbonIndexUtil.isIndexTableExists(carbonTable)
```

Comment:
this is confusing can you add comment ? index is not an index table ?

Reply:
Added